src/arithmetic/vector_funcs/vector_funcs.c

```diff
@@ -12,7 +12,7 @@
 #include "../../datatypes/vector.h"
 
 // create a vector from an array of floats
-// vector32f([0.2, 0.12, 0.3178])
+// vecf32([0.2, 0.12, 0.3178])
```

Comment:
Isn't it going to break clients existing code ?

Reply:
nope, this comment was wrong.